app/src/main/java/com/github/se/gomeet/MainActivity.kt

```diff
@@ -67,10 +66,10 @@ class MainActivity : ComponentActivity() {
             composable(Route.CREATE) { Create(navAction) }
             composable(Route.PROFILE) { Profile(navAction) }
             composable(Route.PRIVATE_CREATE) {
-              PrivateCreate(navAction, EventViewModel(userIdState.value))
+              CreateEvent(navAction, EventViewModel(userIdState.value), true)
             }
             composable(Route.PUBLIC_CREATE) {
-              PublicCreate(navAction, EventViewModel(userIdState.value))
```

Comment:
Very great Idea to merge PrivateCreate and PublicCreate files, it avoids a lot of duplicate code!